plugin/hover.py

```diff
@@ -369,10 +369,10 @@ def on_select(targets: List[str], idx: int) -> None:
                 position = {"line": row, "character": col_utf16}  # type: Position
                 r = {"start": position, "end": position}  # type: Range
                 sublime.set_timeout_async(partial(session.open_uri_async, uri, r))
-        elif urlparse(href).scheme.lower() not in ("", "http", "https"):
-            sublime.set_timeout_async(partial(self.try_open_custom_uri_async, href))
-        else:
+        elif urlparse(href).scheme.lower() in ("", "http", "https"):
```

Comment:
Is there a reason to open the browser when `""`

Reply:
this message is not that important, as you didn't change this line of code...
I wanted just to recall why it was an empty string..

Reply:
I think it is for something like `www.example.com/index.html`. Here the scheme is `""`, but it is likely meant to be a website URL. For local files I guess it can be assumed that they are always absolute paths and have a `file` scheme.